Makefile

```diff
@@ -222,4 +231,39 @@ endif
 
 # Test all
 .PHONY: test-all
-test-all: test-account-creation test-ccs test-reuse test-awsfederatedaccountaccess
\ No newline at end of file
+test-all: test-account-creation test-ccs test-reuse test-awsfederatedaccountaccess
+
+# Test apply ou map cr
+.PHONY: create-ou-map
+create-ou-map:
+	# Create OU map
+	@oc process -p ROOT=${OSD_STAGING_1_OU_ROOT_ID} -p BASE=${OSD_STAGING_1_OU_BASE_ID} -f hack/templates/aws_v1alpha1_configmap_ou_mapping.tmpl | oc apply -f -
+
+# Test delete ou map cr
+.PHONY: delete-ou-map
+delete-ou-map:
+# Delete OU map
+	@oc process -p ROOT=${OSD_STAGING_1_OU_ROOT_ID} -p BASE=${OSD_STAGING_1_OU_BASE_ID} -f hack/templates/aws_v1alpha1_configmap_ou_mapping.tmpl | oc delete -f -
+
+# Test claim-account
+.PHONE: claim-account
```

Comment:
Is this different from the `create-accountclaim` target?

Reply:
yes, `create-accountclaim` uses the `test-claim-namespace` namespace and I am trying to use `aws-account-operator`. I am talking to @jharrington22 about possibly being able to use the target but as of now I don't think it would work.